packages/example/tests/playwright/specs/query-parameters.spec.ts

```diff
@@ -51,29 +58,43 @@ test.describe.parallel('query parameters', () => {
     worker,
   }) => {
     await worker.use(
-      rest.get('/api/search?q=ignoredQuery', (request, response, context) =>
-        response(
-          context.status(200),
-          context.json([
+      http.get(
+        '/api/search?q=ignoredQuery',
+        () =>
+          new Response(
```

Comment:
Is there a reason why you sometimes use the `new Response` and other times the `new HttpResponse` ?

Reply:
Unless you are mocking `Set-Cookie` response header, you should be fine using standard `Response` everywhere. But for compatibility's sake, I recommend using `HttpResponse`. This simply results in a more future-proof solution in case we decide for `HttpResponse` to handle more things in the future. 

Also, here's the [`HttpResponse` vs `Response` explanation](https://mswjs.io/docs/api/http-response#why-not-native-response).

Reply:
I don't have a good reason for doing this. 

Initially, I was in favor of using the standard `Response` class in favor of using native functionality. However the convenience of `HttpResponse.json` was too convenient to not use. 

I intended on circling back and using HttpResponse for all instances, but I didn't get around to it. Thankfully, @mamidenn has submitted a PR to address this :)